environments/azure.yaml

```diff
@@ -232,6 +232,10 @@ admin-ac-tests:
   enabled: true
   testFilter: SmokeTest
 
+of-ac-tests:
+  enabled: true  
```

Comment:
I thought you were turning this off everywhere? Know it should be superseeded by the others, but...